ext/ui-build/data/root.html

```diff
@@ -80,7 +80,7 @@
 		const frame = document.createElement('iframe');
 		frame.name = frameName;
 		frame.style.visibility = 'hidden';
-		frame.allow = 'microphone *; camera *;';
+		frame.allow = 'microphone *; camera *; midi *;';
```

Comment:
This change seems unrelated to the steam api change. Please don't add it in the same pull request.